src/main/resources/db/migration/V1_1__insert_user_admin.sql

```diff
@@ -1,4 +1,4 @@
-INSERT INTO users (id, name, email, password, role)
+INSERT INTO users (id, first_name, email, password, role)
 VALUES ('88596531-7f0f-407d-b502-31833b8c8e8d', 'root', 'root@gmail.com', '$2a$12$fGuoN79WFwHPUmirHOlxIO9kdmMTBrlNGKob0ay4muxXNDePg38ri', 'ADMIN')
 ON CONFLICT (email) DO UPDATE
-SET name = 'root', email = 'root@gmail.com', password = '$2a$12$fGuoN79WFwHPUmirHOlxIO9kdmMTBrlNGKob0ay4muxXNDePg38ri', role = 'ADMIN';
\ No newline at end of file
+SET first_name = 'root', email = 'root@gmail.com', password = '$2a$12$fGuoN79WFwHPUmirHOlxIO9kdmMTBrlNGKob0ay4muxXNDePg38ri', role = 'ADMIN';
```

Comment:
I think we can even skip the firstName of the root user.

Reply:
Actually both firstName and lastName are required if I remember correctly? But this migration is obsolete since we talked about modifying the flyway migrations. After we merge his code onto test, I will merge test into my branch and will create a new migration for the User extension logic. The creation of the root user is handled by Ignat

Reply:
They were not required in the version of the code you reviewed. Fixed.